lms/djangoapps/certificates/views/webview.py

```diff
@@ -97,7 +97,7 @@ def _update_certificate_context(context, user_certificate, platform_name):
 
     # Translators:  The format of the date includes the full name of the month
     context['certificate_date_issued'] = _('{month} {day}, {year}').format(
-        month=user_certificate.modified_date.strftime("%B"),
+        month=_(user_certificate.modified_date.strftime("%B")),
```

Comment:
@amangano-edx  Actually it works. First we get the value of `user_certificate.modified_date.strftime("%B")` and then, the transaltion of that value. We know for sure that `%B` will return the mont's full name, in this case, in English. So, for example, if the month is January, `gettext` will receive that value: `month=_('January'),` and will return the translation, in our example in Spanish: `month='Enero',`.